numba_rvsdg/rendering/rendering.py

```diff
@@ -37,7 +37,7 @@ def render_region_block(
             if regionblock.kind == "head":
                 color = "red"
             subg.attr(color=color, label=regionblock.kind)
-            for name, block in graph.items():
+            for name, block in sorted(graph.items()):
```

Comment:
Subgraphs are initialized by iterating over a set, this uses insertion order to iterate.